Resources/Prototypes/_White/tags.yml

```
@@ -2,4 +2,7 @@
   id: EnergySword
 
 - type: Tag
-  id: NeuroStabilization
\ No newline at end of file
+  id: NeuroStabilization
+
+- type: Tag
+  id: MindSlave
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)